magefile.go

```diff
@@ -2085,3 +2097,12 @@ func hasSnapshotEnv() bool {
 	b, _ := strconv.ParseBool(snapshot)
 	return b
 }
+
+func hasRunUntilFailure() bool {
+	runUntil := os.Getenv("TEST_RUN_UNTIL_FAILURE")
+	if runUntil == "" {
+		return false
+	}
+	b, _ := strconv.ParseBool(runUntil)
+	return b
```

Comment:
[Suggestion]
As you're ignoring the error, you don't need to test for empty string. It'll fail and return false anyway

```suggestion
	b, _ := strconv.ParseBool(runUntil)
	return b
```